browser/css/jsdialogs.css

```diff
@@ -363,6 +364,8 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 	visibility: hidden !important;
 }
 
+#FindReplaceDialog:has(#replaceframe:not(.hidden)) button#replace_tab_btn,
+#findreplacetabbox button#find_tab_btn,
```

Comment:
I think we miss `:has(#replaceframe.hidden)` in the second line

Reply:
The #find_tab_btn is already the default active tab, so line `367` specifically handles the case where the Replace tab should be highlighted based on the visibility of the Replace frame.